pkg/cluster/internal/create/actions/createworker/files/aws/allow-egress-imds_gnetpol.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: crd.projectcalico.org/v1
 kind: GlobalNetworkPolicy
 metadata:
-  name: allow-traffic-to-aws-imds-capa
+  name: allow-traffic-to-aws-imds
```

Comment:
Habrá que tener esto en cuenta para el upgrade, quizá borrando `allow-traffic-to-aws-imds-capa` dado que cambia de nombre